README.rst

```diff
@@ -61,3 +61,72 @@ Related projects
 * `icalevents <https://github.com/irgangla/icalevents>`_. It is built on top of icalendar and allows you to query iCal files and get the events happening on specific dates. It manages recurrent events as well.
 * `recurring-ical-events <https://pypi.org/project/recurring-ical-events/>`_. Library to query an ``ICalendar`` object for events happening at a certain date or within a certain time.
 * `x-wr-timezone <https://pypi.org/project/x-wr-timezone/>`_. Library to make ``ICalendar`` objects and files using the non-standard ``X-WR-TIMEZONE`` compliant with the standard (RFC 5545).
+
+
+Examples
+================
+Create a calendear and add an event to it (master branch)
+
+.. code-block:: python
+
+    #!/usr/bin/env python3
+    from dateutil import tz
```

Comment:
We are already using pytz and hopefully soon using tzinfo, let's not start using dateutil's tz implementation as well.

Also, at least in this snippet, we don't really need it, or am I missing something?